### DISCUSSION TITLE
Gleisangabe und standardmäßiges Anzeigen des Endziels einer (Zug-)Verbindung

### DISCUSSION THREAD

**Discussion by SpDrS423** [Traewelling/traewelling#2127] [Ideas / Feature Requests]
Eine Angabe des Abfahrts- und Ankunftsgleises (sofern vorhanden), sowie eine Anzeige des komplett ersten Starts und letzten Ziels einer Verbindung, z.B. Fahrt mit der S3 von München Pasing nach München Ost:

- S3 (6390) kommt aus Mammendorf
München-Pasing 20:45 Gl. 6
München Ost 20:59 Gl. 4
- weiter nach Holzkirchen

**Comment by pascalwittler** [comment]
Kannst du ggf. einen Screenshot oder ein Beispiel machen, wie die aktuelle Darstellung (an welcher Stelle) aussieht und daneben/darunter, wie du dir die Darstellung vorstellst?

**Comment by MrKrisKrisu** [comment]
In den experimentellen Features gibts jetzt Gleisangaben im Stationboard. In der StatusCard würde ich da eher verzichten, da sie jetzt schon zu voll ist.
